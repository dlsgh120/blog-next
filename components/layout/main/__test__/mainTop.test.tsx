import React from 'react'
import {render} from '@testing-library/react';
import MainTop from '../mainTop';

describe('<MainTop />',()=>{
    
    it('MainTop snapshot',()=>{
        const el = render(<MainTop />);
        expect(el).toMatchSnapshot();
    });

    it('has a logo of <h1> tag from main component', ()=>{
        const el = render(<MainTop />);
        const logo = el.getByText('최인호의 Dev Blog 입니다.');

        expect(logo).toBeTruthy();
    })
});