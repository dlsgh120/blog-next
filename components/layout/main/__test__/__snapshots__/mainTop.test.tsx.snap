// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainTop /> MainTop snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="mainTop__MainBackground-sc-1g8isn9-0 jjuguW"
        style="background-image: url(./images/main_background.jpg);"
      >
        <h1>
          최인호의 Dev Blog 입니다.
        </h1>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="mainTop__MainBackground-sc-1g8isn9-0 jjuguW"
      style="background-image: url(./images/main_background.jpg);"
    >
      <h1>
        최인호의 Dev Blog 입니다.
      </h1>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
